help/transform/missing-values-imputation.md

```diff
@@ -2,31 +2,22 @@
 title: "Missing values imputation"
 ---
 
-Imputation is the process of replacing missing data with substituted values.
+[Imputation](https://en.wikipedia.org/wiki/Imputation_\(statistics\)) is the process of replacing missing data with substituted values.
 
-## Algorithm
-
-Missing values imputation algorithm is based on **k-nearest neighbors algorithm (k-NN)**. This algorithm is a
-non-parametric method used for classification and regression. Both for classification and regression, it can be useful
-to assign weight to the contributions of the neighbors, so that the nearer neighbors contribute more to the average than
-the more distant ones. For example, a common weighting scheme consists in giving each neighbor a weight of **1/d**,
-where **d** is the distance to the neighbor.
+Datagrok imputes missing values using the k-nearest neighbors method ([k-NN](https://public.datagrok.ai/js/samples/domains/data-science/missing-values-imputation)).
 
 ## Run
```

Comment:
It's a very short article. You don't need a separate section for this. Simply write a lead in a sentence like this: To impute missing values:

Reply:
Done